src/layout/dashboard/header/LanguagePopover.js

```diff
@@ -35,7 +37,7 @@ export default function LanguagePopover() {
             option.value === language && (
               <Image
                 key={option.id}
-                src={option.icon || DEFAULT_IMAGE}
+                src={option.icon2 || DEFAULT_IMAGE}
```

Comment:
rename field for its usage like image, not meaningless name

Reply:
DEFAULT_IMAGE ?

Reply:
icon2? but you define its value as an image

Reply:
I have changed the icon - circleIcon and icon2 - flagIcon